@here/harp-mapview/lib/MapView.ts

```diff
@@ -3313,6 +3440,29 @@ export class MapView extends THREE.EventDispatcher {
                     return;
                 }
                 this.m_scene.add(light);
+                if ((light as any).isDirectionalLight) {
+                    const directionalLight = light as THREE.DirectionalLight;
+                    // This is needed so that the target is updated automatically, see:
+                    // https://threejs.org/docs/#api/en/lights/DirectionalLight.target
+                    this.m_scene.add(directionalLight.target);
+
+                    this.m_directionalLight = directionalLight;
```

Comment:
There could be multiple directional light. You just pick the last one but I don't see any reason why multiple directional lights would not work (except for performance)